lib/postcss-sorting.coffee

```diff
@@ -19,12 +19,33 @@ module.exports =
       description: 'Display notification on successful sort.'
       type: 'boolean'
       default: true
-
+    shouldUpdateOnSave:
+      title: 'On Save'
+      description: 'Process file on every save.'
+      type: 'boolean'
+      default: true
```

Comment:
@paweljedrzejczyk, I don't like this "enabled by default" feature, because (opposite to notifications option) it will affect existing users, who don't want to process on save 😞